lib/modules/datasource/hex/index.ts

```diff
@@ -18,6 +18,13 @@ export class HexDatasource extends Datasource {
 
   override readonly defaultVersioning = hexVersioning.id;
 
+  override readonly releaseTimestampSupport = true;
+  override readonly releaseTimestampNote =
+    'The relase timestamp is determined the `inserted_at` field in the results.';
```

Comment:
```suggestion
    'The release timestamp is determined from the `inserted_at` field in the results.';
```

Typo in the word `release`. And a missing word `from`.